docs/overview/office-add-in-code-samples.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Office Add-in code samples
+description: 'Office Add-in code samples'
+ms.date: 09/09/2021
+localization_priority: Normal
+---
+
+# Office Add-in code samples
+
+The code samples listed in this article are written to help you learn how to use various features when developing Office Add-ins.
+
+## Outlook
+
+| Name                | Description         |
+|:--------------------|:--------------------|
+| [Use Outlook event-based activation to tag external recipients (preview)](/samples/officedev/pnp-officeaddins/outlook-add-in-tag-external-recipients) | This sample uses event-based activation to run an Outlook add-in when the user changes recipients while composing a message. The add-in also uses the appendOnSendAsync API to add a disclaimer. |
+| [Use Outlook event-based activation to set the signature](/samples/officedev/pnp-officeaddins/outlook-add-in-set-signature/) | This sample uses event-based activation to run an Outlook add-in when the user creates a new message or appointment. The add-in can respond to events, even when the task pane is not open. It also uses the setSignatureAsync API. |
+
+## Excel
+
+| Name                | Description         |
+|:--------------------|:--------------------|
+| [Open in Teams](/samples/officedev/pnp-officeaddins/office-excel-add-in-open-in-teams/) | Creates a new Excel spreadsheet in Microsoft Teams containing data you define.|
+| [Create custom contextual tabs on the ribbon](/samples/officedev/pnp-officeaddins/office-add-in-contextual-tabs/) | This sample shows how to create a custom contextual tab on the ribbon in the Office UI. The sample creates a table, and when the user moves the focus inside the table, the custom tab is displayed. When the user moves outside the table, the custom tab is hidden. |
+| [Use keyboard shortcuts for Office add-in actions](/samples/officedev/pnp-officeaddins/office-add-in-keyboard-shortcuts) | Shows how to set up a basic Excel add-in project that utilizes keyboard shortcuts. |
+| [Custom function sample using web worker](/samples/officedev/pnp-officeaddins/excel-custom-function-web-worker-pattern/) | Shows how to use web workers in custom functions to prevent blocking the UI of your Office Add-in. |
```

Comment:
```suggestion
| [Custom function sample using web worker](/samples/officedev/pnp-officeaddins/excel-custom-function-web-worker-pattern/) | Use web workers in custom functions to prevent blocking the UI of your Office Add-in. |
```